app/models/account.rb

```diff
@@ -33,4 +33,28 @@ def member?
   def active_member?
     member? && !member.inactive?
   end
+
+  def cascade_destroy
+    member_id = member.id
+    Position
+      .where(recruiter_id: member_id)
+      .find_each do |position|
+        position.update!(recruiter_id: nil)
+      end
+
+    Scorecard
+      .where(interviewer_id: member_id)
+      .find_each(&:destroy!)
+
+    Event
```

Comment:
Would placements, emails, files, created by this member, be removed? Otherwise, we may got case that placement is still exists, but without added event.

Reply:
We don't have explicit association `accounts` -> `events`, so events' actor_account_ids are not nullified automatically and foreign key constraint violation is raised. So I added an explicit nullification here